services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -152,3 +172,68 @@ async function putRecurringVivaCase(PK, workflowId, period) {
 
   return caseItem;
 }
+
+async function getUser(PK) {
+  const personalNumber = PK.replace('USER#', '');
+  const params = {
+    TableName: config.users.tableName,
+    Key: {
+      personalNumber,
+    },
+  };
+
+  const [error, dbResponse] = await to(dynamoDB.call('get', params));
+  if (!dbResponse) {
+    throwError(error.statusCode, error.message);
+  }
```

Comment:
same feedback as previous comments on how error handling is done in other lambdas
check IF error is a little more telling :)